bin/download_extract_rc.ps1

```diff
@@ -0,0 +1,61 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+# Helper script for lazy Windows Developers
+# Downloading, checking and extracting the sources of a rc candidate for Windows
+
+$DOWNLOAD_VERSION = "3.4.2"

```

Comment:
Are these variables meant to be parametric?  It would be nice if the script could take them as external parameters.

Reply:
Good idea, not my first idea for a first version ;-), but added some parameters!